client/views/chat.tpl

```diff
@@ -2,7 +2,11 @@
 <div id="chan-{{id}}" data-title="{{name}}" data-id="{{id}}" data-type="{{type}}" class="chan {{type}}">
 	<div class="header">
 		<button class="lt"></button>
-		<button class="rt"></button>
+		{{#equal type "lobby"}}
+			{{!-- Insert settings icon --}}
+		{{else}}
+			<button class="rt"></button>
+		{{/equal}}
```

Comment:
Actually, checking that the type is a channel is probably more appropriate to make sure it doesn't show up on lobbies and private messages. Anything else we should display or not display this?